patronload/patron.py

```diff
@@ -62,6 +67,152 @@ def populate_patron_common_fields(
+    """
+    patron_template.first_name.string = (  # type: ignore[union-attr]
+        patron_dict["FIRST_NAME"] or ""
+    )
+    patron_template.middle_name.string = (  # type: ignore[union-attr]
+        patron_dict["MIDDLE_NAME"] or ""
+    )
+    patron_template.last_name.string = (  # type: ignore[union-attr]
+        patron_dict["LAST_NAME"] or ""
+    )
+
+    patron_template.address.line1.string = (  # type: ignore[union-attr]
+        patron_dict["TERM_STREET1"]
+        if patron_dict["TERM_STREET1"]
+        else "NO ADDRESS ON FILE IN DATA WAREHOUSE"
+    )
+    patron_template.line3.string = (  # type: ignore[union-attr]
+        patron_dict["TERM_STREET2"] or ""
+    )
+    patron_template.city.string = (  # type: ignore[union-attr]
+        patron_dict["TERM_CITY"] or ""
+    )
+    patron_template.state_province.string = (  # type: ignore[union-attr]
+        patron_dict["TERM_STATE"] or ""
+    )
+    patron_template.postal_code.string = (  # type: ignore[union-attr]
+        patron_dict["TERM_ZIP"] or ""
+    )
+
+    office_phone_number = patron_template.find_all("phone_number")[0]
+    home_phone_number = patron_template.find_all("phone_number")[1]
+    if patron_dict["OFFICE_PHONE"] and patron_dict["TERM_PHONE1"]:
+        office_phone_number.string = format_phone_number(patron_dict["OFFICE_PHONE"])
+        home_phone_number.string = format_phone_number(patron_dict["TERM_PHONE1"])
+    elif (
+        patron_dict["OFFICE_PHONE"]
+        or patron_dict["TERM_PHONE1"]
+        or patron_dict["TERM_PHONE2"]
+    ):
+        home_phone_number.parent.decompose()
+        office_phone_number.string = (
+            format_phone_number(patron_dict["OFFICE_PHONE"])
+            if patron_dict["OFFICE_PHONE"]
+            else (
+                format_phone_number(patron_dict["TERM_PHONE1"])
+                if patron_dict["TERM_PHONE1"]
+                else format_phone_number(patron_dict["TERM_PHONE2"])
+            )
+        )
+    else:
+        patron_template.phones.clear()  # type: ignore[union-attr]
+
+    if patron_dict["HOME_DEPARTMENT"] in STUDENT_DEPARTMENTS:
+        patron_template.statistic_category.string = (  # type: ignore[union-attr]
+            STUDENT_DEPARTMENTS[patron_dict["HOME_DEPARTMENT"]]
+        )
+    else:
+        patron_template.statistic_category.string = "ZZ"  # type: ignore[union-attr]
+        logger.error("Unknown dept: %s", patron_dict["HOME_DEPARTMENT"])
+
+    # User Group codes
+    # 31 = Student - Undergraduate
+    # 32 = Student - Graduate
+    # 54 = Non-MIT - Cross-registered
+    if patron_dict["STUDENT_YEAR"]:
+        if re.search("^[1234Uu]$", patron_dict["STUDENT_YEAR"]):
+            patron_template.user_group.string = "31"  # type: ignore[union-attr]
+        elif re.search("^[Gg]$", patron_dict["STUDENT_YEAR"]):
+            patron_template.user_group.string = "32"  # type: ignore[union-attr]
+        if re.search("^NI[UVWTRH]$", patron_dict["HOME_DEPARTMENT"]):
```

Comment:
Looking at the original code logic, I think this should be unindented one level.